cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/MessagesApiModelExt.kt

```diff
@@ -2,11 +2,52 @@ package com.sourcepoint.cmplibrary.data.network.model.optimized
 
 import com.sourcepoint.cmplibrary.data.network.converter.JsonConverter
 import com.sourcepoint.cmplibrary.data.network.converter.converter
+import com.sourcepoint.cmplibrary.data.network.model.optimized.includeData.IncludeDataGppParam
+import com.sourcepoint.cmplibrary.data.network.model.optimized.messages.OperatingSystemInfoParam
+import com.sourcepoint.cmplibrary.data.network.util.CampaignsEnv
 import com.sourcepoint.cmplibrary.exception.CampaignType
 import com.sourcepoint.cmplibrary.model.CampaignReq
 import kotlinx.serialization.encodeToString
 import kotlinx.serialization.json.* // ktlint-disable
 
+internal fun getMessageBody(
+    propertyHref: String,
+    accountId: Long,
+    campaigns: List<CampaignReq>,
+    cs: ConsentStatus?,
+    ccpaStatus: String?,
+    consentLanguage: String?,
+    campaignEnv: CampaignsEnv?,
+
+    os: OperatingSystemInfoParam = OperatingSystemInfoParam()
+): JsonObject {
+    return buildJsonObject {
+        put("accountId", accountId)
+        campaignEnv?.env?.let { put("campaignEnv", it) }
+        putJsonObject("includeData") {
+            putJsonObject("TCData") {
+                put("type", "RecordString")
+            }
+            putJsonObject("campaigns") {
+                put("type", "RecordString")
+            }
+            putJsonObject("webConsentPayload") {
+                put("type", "RecordString")
+            }
+            // TODO
+            put("", JsonConverter.converter.encodeToJsonElement(IncludeDataGppParam()))
```

Comment:
@bohdan-go-wombat This is the point that needs refactored, we need to re-add the gpp include data.
For now just reconstruct the json, in the next step we can refactor it